sdk/samples/iot/README.md

```diff
@@ -74,25 +74,59 @@ This section provides an overview of the different samples available to run and
+  This [sample](https://github.com/Azure/azure-sdk-for-c/blob/master/sdk/samples/iot/paho_iot_hub_pnp_sample.c) connects an IoT Plug and Play enabled device with the Digital Twin Model ID (DTMI) detailed [here](https://github.com/Azure/opendigitaltwins-dtdl/blob/master/DTDL/v2/samples/Thermostat.json). If a timeout occurs while waiting for a message from the Azure IoT Explorer, the sample will exit. X509 self-certification is used.
 
-  In short, the capabilities are listed here:
-- **Methods**: Invoke a method called `getMaxMinReport` with JSON payload value `"since"` with an [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) value for start time for the report. The method sends a response containing the following JSON payload:
+  To interact with this sample, **you must use the Azure IoT Explorer**. The capabilities are listed below:
+
+- **Device Twin**: Two device twin properties are supported in this sample.
+  - A desired property named `targetTemperature` with a `double` value for the desired temperature.
+  - A reported property named `maxTempSinceLastReboot` with a `double` value for the highest temperature.
+
+  To send a device twin desired property message, select your device's Device Twin tab in the Azure IoT Explorer. Add the property `targetTemperature` along with a corresponding value to the `desired` section of the JSON. Select Save to update the twin document and send the twin message to the device.
 
   ```json
-  {
-    "maxTemp": 20,
-    "minTemp": 20,
-    "avgTemp": 20,
-    "startTime": "<ISO8601 time>",
-    "endTime": "<ISO8601 time>"
+  "properties": {
+      "desired": {
+          "targetTemperature": 68.5,
+      }
   }
   ```
 
-  with correct values substituted for each field.
+  Upon receiving a desired property message, the sample will update the twin property locally and send a reported property of the same name back to the service. This message will include a set of "ack" values: `ac` for the HTTP-like ack code, `av` for ack version of the property, and an optional `ad` for an ack description.
+
+  ```json
+  "properties": {
+      "reported": {
+          "targetTemperature": {
+            "value": 68.5,
+            "ac": 200,
+            "av": 14,
+            "ad": "success"
+          },
+          "maxTempSinceLastReboot": 74.3,
+      }
+  }
+  ```
+
+- **Direct Method (Command)**: One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an [ISO8061](https://en.wikipedia.org/wiki/ISO_8601) time format and select Invoke method.
```

Comment:
```suggestion
- **Direct Method (Command)**: One method is supported in this sample: `getMaxMinReport`. If any other methods are attempted to be invoked, the log will report the method is not found. To invoke a method, select your device's Direct Method tab in the Azure IoT Explorer. Enter the method name `getMaxMinReport` along with a payload using an [ISO8601](https://en.wikipedia.org/wiki/ISO_8601) time format and select Invoke method.
```

Reply:
me and typos.  maybe I started too early, or stayed up too late on this.